tee-worker/ts-tests/integration-tests/common/utils/identity-helper.ts

```diff
@@ -35,11 +35,11 @@ export async function buildIdentityHelper(
     const identity = {
         [type]: address,
     };
-
     const encodedIdentity = context.sidechainRegistry.createType(
         'LitentryPrimitivesIdentity',
         identity
     ) as unknown as LitentryPrimitivesIdentity;
```

Comment:
I think this `as` should no longer be needed, if `sidechainRegistry` has the correct typing now

Reply:
Let's make the modifications for this in another pull request because there are many similar pieces of code. We can't wait any longer for this pull request. :p

Reply:
will open a followup ticket to resolve together with below one

Reply:
I think we can improve it in #1878